scss/utilities/_inputs.scss

```diff
@@ -182,3 +182,7 @@ input:disabled ~ label,
 textarea:disabled ~ .form-text {
     font-style: italic;
 }
+
+input[readonly], textarea[readonly] {
+    background-color: var(--bs-tertiary-bg);
```

Comment:
```suggestion
    background-color: $input-disabled-bg;
```

Let's use the sass variable just incase we ever change the color of the of the disabled elements, it'll automatically change here too.